contracts/CompoundGovernor.sol

```diff
@@ -135,32 +135,74 @@ contract CompoundGovernor is
         return GovernorSequentialProposalIdUpgradeable.hashProposal(_targets, _values, _calldatas, _descriptionHash);
     }
 
+    /// @notice Creates a new proposal.
+    /// @dev This is an override of the function that skips proposal threshold check for whitelisted accounts.
```

Comment:
this sounds like it's referring to some other function (in GovernorUpgradeable) that skips the proposal threshold check. but in fact, THIS is the function doing that.

i think i'd recommend expanding the @notice to include details on how the whitelisting works as well, otherwise that'll be undocumented afaik

Reply:
Updated in:  https://github.com/ScopeLift/compound-governance-upgrade/pull/89/commits/f6bb8a8962822ec04012400346e884d57f9def4c and added to the whitelisting related docs above `setWhitelistAccountExpiration`.